data/hai/unfettered jobs.txt

```diff
@@ -0,0 +1,479 @@
+	job
+	repeat
+	description "Collect a tribute payment of <tons> of food from <stopovers>, and bring it to the Unfettered on <destination>. Your share of the tribute will be <payment>."
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		random < 60
+	cargo "food (tribute)" 35 2 .05
+	on complete
+		payment 7000 2000
+		dialog phrase "unfettered tribute payment dialog"
+	source
+		attributes "unfettered"
+	stopover
+		distance 4 5
+		attributes "hai"
+		attributes "spaceport"
+	on visit
+		dialog phrase "generic missing stopover or cargo"
+	on stopover
+		dialog phrase "unfettered tribute pickup dialog"
+
+mission "Unfettered Tribute 3"
+	name "Hai Tribute to <planet>"
+	job
+	repeat
+	description "Collect a tribute payment of <tons> of food from <stopovers>, and bring it to the Unfettered on <destination>. Your share of the tribute will be <payment>."
+	to offer
+		has "Unfettered Jump Drive 2: offered"
+		random < 40
+	cargo "food (tribute)" 45 2 .05
+	on complete
+		payment 9000 2400
+		dialog phrase "unfettered tribute payment dialog"
+	source
+		attributes "unfettered"
+	stopover
+		distance 5 6
+		attributes "hai"
+		attributes "spaceport"
+	on visit
+		dialog phrase "generic missing stopover or cargo"
+	on stopover
+		dialog phrase "unfettered tribute pickup dialog"
+
+
+
+mission "Proud Unfettered Boarded"
+	boarding
+	repeat
+	minor
+	to offer
+		"flagship crew" > 120
+		random < "flagship crew" / 8
+	source
+		government "Hai (Unfettered)"
+		category "Heavy Warship"
+	on offer
+		conversation
+			`As you board the <origin> you immediately notice something is wrong: the ship is starting to explode! The captain sends you a message. "This is not a fair fight. It is better to die with my crew standing than risk being slaughtered by inferior humans! You will not be allowed to capture this ship and use it against our brothers and sisters. I will take you down with us!"`
+				flee
+	destination "Darkcloak"
+
+
+
+mission "Unfettered Silicon Asteroid Mining"
+	name `Gather silicon`
+	description `Mine and then collect 50 tons of silicon from the asteroid field in Wah Yoot, and then bring them back to <destination> for <payment>.`
+	job
+	repeat
+	source "Darkcloak"
+	to offer
+		has "First Contact: Unfettered: declined"
+		random < 60
+	on complete
+		payment 250000
+		dialog `The group of Unfettered unloading the materials you mined seems enthused. "Keep this up and we will be building new ships in no time! Some use those asteroids as target practice, you know, but most find it dull and too easy to fight an enemy that does not shoot or react in any other way," one tells you as it hands over your payment of <payment> credits.`
+		outfit "Silicon" -50
+
+mission "Unfettered Silver Asteroid Mining"
+	name `Gather silver from asteroids`
+	description `Mine and then collect 15 tons of silver from the asteroid field in Ehma Ti and then bring them back to <destination> for <payment>.`
+	job
+	repeat
+	source "Firelode"
```

Comment:
If I remember correctly the silver is in the system of Firelode
Currently they ask you to mine in the same system you get the mission at

Reply:
So for the silicon mission: Offered on Darkcloak, mine in system Darkcloak is in, return to Darkcloak.
For the silver mission: Offered on Firelode, mine in system Firelode is in, return to Darkcloak.
Obviously that implies that most of the production is taking place on Darkcloak. Why wouldn't Darkcloak offer the same job since they're the ones that need the silver?